xclim/indices/_multivariate.py

```diff
@@ -538,6 +539,72 @@ def liquid_precip_ratio(
     return ratio
 
 
+@declare_units("mm", pr="[precipitation]", tas="[temperature]")
+def precip_accumulation(
+    pr: xr.DataArray,
+    tas: xr.DataArray = None,
+    phase: str = "both",
+    freq: Optional[str] = None,
+) -> xr.DataArray:
+    r"""Accumulated total (liquid and/or solid) precipitation.
+
+    Resample the original daily mean precipitation flux and accumulate over each period.
+    If the daily mean temperature is provided, the phase keyword can be used to only sum precipitation of a certain phase.
+    When the mean temperature is over 0 degC, precipitatio is assumed to be liquid rain and snow otherwise.
+
+    Parameters
+    ----------
+    pr : xr.DataArray
+      Mean daily precipitation flux [Kg m-2 s-1] or [mm].
+    tas : xr.DataArray, optional
+      Mean daily temperature [℃] or [K]
+    phase : str, optional,
+      Which phase to consider, "liquid" or "solid", if None (default), both are considered.
+    freq : str, optional
+      Resampling frequency as defined in
+      http://pandas.pydata.org/pandas-docs/stable/timeseries.html#resampling. Defaults to "YS"
+
+    Returns
+    -------
+    xr.DataArray
+      The total daily precipitation at the given time frequency for the given phase.
+
+    Notes
+    -----
+    Let :math:`PR_i` be the mean daily precipitation of day :math:`i`, then for a period :math:`j` starting at
+    day :math:`a` and finishing on day :math:`b`:
+
+    .. math::
+
+       PR_{ij} = \sum_{i=a}^{b} PR_i
+
+    If `phase` is "liquid", only times where the daily mean temperature :math:`T_i` is above or equal to 0 °C are considered, inversely for "solid".
+
+    Examples
+    --------
+    The following would compute for each grid cell of file `pr_day.nc` the total
+    precipitation at the seasonal frequency, ie DJF, MAM, JJA, SON, DJF, etc.:
+
+    >>> pr_day = xr.open_dataset('pr_day.nc').pr
+    >>> prcp_tot_seasonal = precip_accumulation(pr_day, freq="QS-DEC")
+    """
+    freq = freq or "YS"
+
+    if phase != "both":
```

Comment:
"both" or None ?

Reply:
This is a choice that we could make. What do you prefer?
For me "both" was more explicit.

Reply:
Not a strong opinion on this. I think `None` is more universal.  @tlogan2000 ?

Reply:
No strong opiinon either but I would maybe go with ```None``` to be consistent with some other default behavoir in xclim as a whole

Reply:
2 vs 1, allons-y.